api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceIntegrationTests.java

```diff
@@ -121,6 +122,38 @@ void shouldGetClaimById() {
         assertThat(claimResponse.getClaimant().getAddress()).isEqualToComparingFieldByField(claim.getClaimant().getAddress());
     }
 
+    @Test
+    void shouldGetAllClaims() {
+        // Given
+        Claim homerClaim = aValidClaimWithNinoAndRefernce(HOMER_NINO, HOMER_CLAIM_REFERENCE);
+        claimRepository.saveAll(List.of(homerClaim));
+        List<ClaimResponseDTO> claimResponseDTO = claimToClaimResponseDTOConverter.convert(List.of(homerClaim));
+
+        // When
+        ResponseEntity<List<ClaimResponseDTO>> response =
+                restTemplate.exchange(buildRetrieveAllClaimEntity(), new ParameterizedTypeReference<List<ClaimResponseDTO>>() {});
+
+        // Then
+        List<ClaimResponseDTO> claimResponse = response.getBody();
+        assertThat(response.getStatusCode()).isEqualTo(OK);
+        assertThat(claimResponse).isNotNull();
+        assertThat(claimResponse).hasSameSizeAs(List.of(homerClaim));
+        assertThat(claimResponse.get(0)).isEqualToComparingOnlyGivenFields(claimResponseDTO.get(0),
+                "id", "claimStatus", "firstName", "lastName", "dateOfBirth", "addressLine1", "postcode", "reference");
+        assertThat(claimResponse.get(0).getFirstName()).isEqualTo(homerClaim.getClaimant().getFirstName());
+        assertThat(claimResponse.get(0).getReference()).isEqualTo(homerClaim.getReference());
+    }
+
+    @Test
+    void shouldReturnAnEmptyListWhenNoClaimsExists() {
+        //when
+        ResponseEntity<List<ClaimResponseDTO>> response =
+                restTemplate.exchange(buildRetrieveAllClaimEntity(), new ParameterizedTypeReference<List<ClaimResponseDTO>>() {});
+
+        //then
+        assertThat(response.getBody()).isEmpty();
+    }
+
     @Test
```

Comment:
Should we put a comment here so that we know which tests are for which end points, or split tests in different classes